installers/ansible/roles/pgo-operator/files/pgo-backrest-repo/sshd_config

```diff
@@ -125,3 +125,6 @@ Subsystem	sftp	/usr/libexec/openssh/sftp-server
 #	AllowTcpForwarding no
 #	PermitTTY no
 #	ForceCommand cvs server
+
+# ensure nss_wrapper env vars are set when executing commands as needed for OpenShift compatibility
+ForceCommand NSS_WRAPPER_SUBDIR=ssh . /opt/crunchy/bin/nss_wrapper_env.sh && $SSH_ORIGINAL_COMMAND
```

Comment:
Feels like we need to note that existing clusters need to reload their sshd_config? I believe this is recreated as part of the upgrade process?

Reply:
Another [commit](https://github.com/CrunchyData/postgres-operator/pull/2356/commits/1bfab07ac063c506510342f24755b5e42c0e5ba7) has been pushed to address upgrade scenarios (it turns out that the `recreateBackrestRepoSecret()` function does not actually recreate the `sshd_config` in the pgBackRest repo secret).

In testing this I also found that the recent change to `UsePAM` was not being brought over during the upgrade, so the commit addresses that issue as well (i.e. it ensures `UsePAM` is set to `no`).